src/components/app-configure/auth/AuthProviderInput.vue

```diff
@@ -16,8 +22,23 @@ defineProps({
   },
 })
 
+const toast = useToast()
+const appId = useAppId()
+const appsStore = useAppsStore()
+const app = appsStore.app(appId)
+
+const redirectUri = `https://oauth.arcana.network/auth/apple/redirect/${app.address}`
```

Comment:
This url works for testnet and mainnet?

Reply:
no this is different for dev, testnet and mainnet